app/views/grade_entry_forms/_react_grades_table.js.jsx.erb

```diff
@@ -154,11 +154,7 @@
           this.setState({
             error: error
           });
-        }.bind(this),
-        // Close working indicator.
-        complete: function () {
-          document.getElementById('working').style.display = 'none';
-        }
```

Comment:
Deleted the `complete` function since we don't want the "Loading" popup to go away just yet. This will happen on line 178 in the next ajax call.